sonar-project.properties

```diff
@@ -0,0 +1,5 @@
+sonar.projectKey=omise_omise-ruby
+sonar.organization=omise
+sonar.c.file.suffixes=-
+sonar.cpp.file.suffixes=-
+sonar.objc.file.suffixes=-
```

Comment:
Do we need to [exclude test ](https://github.com/omise/omise-php/blob/master/sonar-project.properties#L5)folder?
`sonar.exclusions=**/test/**`

Reply:
@danfowler I am curious why we are adding properties for C, C++ and Obj C here?

Reply:
@ajzkk @aashishgurung Thanks, resolved both.

Reply:
@aashishgurung I originally added those properties because I was getting an error in the logs that suggested setting those properties to silence them, but now it appears the error is no longer occurring.